packages/blade/src/components/TopNav/_decisions/decisions.md

```diff
@@ -122,6 +122,157 @@ The container for the right-aligned actions like search, buttons, and user profi
 
 ### TabNav
 
+The horizontal navigation bar that contains multiple navigation items, each represented by TabNavItem. 
+
+TabNav's interaction is as follows: 
+
+1. There might already be items under More even at the highest screen size. These items might need a custom order.
+2. Once the screen size starts getting reduced, more items will start moving into More.
+3. The order of these items would follow a logical order. They keep getting added ahead of the already existing items. So the last item to go into More is the first item that you see when you click on More.
+
+TabNav automatically handles all these edge cases & responsiveness.
+
+```jsx
+<TabNav
+  items={[
+    { href: '/home', title: 'Home' },
+    { href: '/payroll', title: 'Payroll' },
+    { href: '/payments', title: 'Payments' },
+    {
+      href: '/rize',
+      title: 'rize',
+      description: 'rize description',
+      icon: RizeIcon,
+      // force this item to always be inside "more" regardless of screen size
+      isAlwaysInMore: true,
+    },
+  ]}
+>
+  {({ items, moreItems }) => {
+    return (
+      <>
+        <TabNavItems>
+          {items.map((item) => (
+            <TabNavItem href={item.href}>{item.title}</TabNavItem>
+          ))}
+        </TabNavItems>
+        <Menu>
+          <TabNavItem>More</TabNavItem>
+          {moreItems.map((item) => {
+            return (
+              <MenuItem key={item.title}>
+                <CustomExploreItem
+                  icon={item.icon}
+                  title={item.title}
+                  description={item.description}
+                />
+              </MenuItem>
+            );
+          })}
+        </Menu>
+      </>
+    );
+  }}
+</TabNav>
+```
+
+#### TabNav Props
+
+Main tab navigation component that will contain multiple navigation items and will handle the responsiveness of the items.
+
+```ts
+type TabNav = {
+  children: React.ReactNode;
+  /**
+   * Array of navigation items
+   */
+  items: Array<
+    TabNavItem & {
+      /**
+       * force this item to always be inside "more" regardless of screen size
+       */
+      isAlwaysInMore?: boolean;
```

Comment:
`isAlwaysInMore` this might not be scalable because we can change the title to "Explore" so my proposal would be to name it generic eg: `showAlwaysInOverflowMenu` or if you have something better

Reply:
Right.

Reply:
Should we name it `isAlwaysOverflowing` since calling it "isAlwaysInOverflowMenu" also makes it a bit tied to the "menu" component. 🤔

Reply:
Going ahead with `isAlwaysOverflowing`